pyaedt/workflows/templates/Run_PyAEDT_Toolkit_Script.py_build

```diff
@@ -1,66 +1,69 @@
-# -*- coding: utf-8 -*-
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
 """
 * * * This script is meant to run in IronPython within AEDT. * * *
-The script provides for choosing the Python script to execute.
-
-It looks for a reference to a Python interpreter in the ``python_interpreter.bat`` file.
 
-It then uses this Python interpreter to execute the script.
-See the declaration of the command variable to see the order in which arguments are passed to the script.
-
-The commands allow the launched script to still reference the project and design that was active when the script
-was launched as well as the AEDT instance that has them open.
+The script allows to execute PyAEDT workflow.
 
 """
 import os
 import sys
 
-from System.Windows.Forms import MessageBox
-from System.Windows.Forms import MessageBoxButtons
-from System.Windows.Forms import MessageBoxIcon
-
 is_linux = os.name == "posix"
-script_name = os.path.splitext(os.path.basename(__file__))[0]
 
 if is_linux:
     import subprocessdotnet as subprocess
 
+toolkits_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+
+sys.path.append(toolkits_dir)
+
+import PyAEDT_utils
+
+
 def main():
     try:
+        # Get AEDT version
+        short_version = oDesktop.GetVersion()[2:6].replace(".", "")
         oDesktop.AddMessage("", "", 0, "Toolkit launched. Please wait.")
```

Comment:
```suggestion
        oDesktop.AddMessage("", "", 0, "Toolkit launched. Wait.")
```
Omit "please" in technical doc. It might be nice if this message indicated what the wait was for. Or, perhaps, change it to "Toolkit is launching...."